packages/react/src/schematics/application/application.ts

```diff
@@ -76,7 +74,6 @@ export default function(schema: Schema): Rule {
       updateJestConfig(options),
       addStyledModuleDependencies(options),
       addRouting(options, context),
-      addBabelPresetForReact(options),
```

Comment:
did we remove it because it's not needed directly by the generated app, but rather by our builder? 

i.e. it will be brought in by our builder, no need to have it on the root `package.json` of the generated app?

Reply:
I moved it to a `dependency` of `@nrwl/react` instead since other babel packages are `dependencies` of `@nrwl/web`, this will make things more consistent. It also means we can fully control the versions of the packages easier (without having to migrate).

Reply:
makes sense, thanks!